chart/values.yaml

```diff
@@ -86,3 +86,5 @@ csi:
     requests:
       cpu: "16m"
       memory: "64Mi"
+
+namespace: "mayastor"
```

Comment:
do we need this, can't we use just use "{{ .Release.Namespace }}" ?

Reply:
I've removed this.